src/libraries/System.Private.CoreLib/src/System/Collections/Concurrent/ConcurrentQueueSegment.cs

```diff
@@ -37,6 +37,9 @@ internal sealed class ConcurrentQueueSegment<T>
         internal ConcurrentQueueSegment<T>? _nextSegment; // SOS's ThreadPool command depends on this name
 #pragma warning restore 0649
 
+        /// <summary>Threshold to spin before allowing threads to sleep when there is contention.</summary>
+        internal const int _sleep1Threshold = 8;
```

Comment:
Nit, consts are PascalCase in our style. Also can this be private?

Reply:
Yep! Updated to PascalCase and private.